newsrevenuehub/rev-engine
# Nrh 180  sidebar blurb

#### What's this PR do?
Add the ability to create sidebar blurbs. Currently the supported blocks are RichText and Images.  Once #116 is merged we can add in benefit levels.

#### How should this be manually tested?

1. Go to any page, and open up the page editor.
2. You will note a new tab `Sidebar`
3. Open `Sidebar` and begin adding elements
4. Reorder as you wish
5. Close the editor and save the page
6. Note the sidebar is displayed as constructed. 

#### Do any changes need to be made before deployment to staging? production? (adding environment variables, for example)?
No.
